frontend/common/useInfiniteScroll.ts

```diff
@@ -70,6 +76,7 @@ const useInfiniteScroll = <
     isLoading: queryResponse.isLoading,
     loadMore,
     loadingCombinedData: loadingCombinedData && queryResponse.isFetching,
+    refetchData,
```

Comment:
I think this might get confusing with refresh. If it's not doing what we want (I guess you want to set page to 1 + refetch) then we should do that I think

Reply:
I corrected the refresh function and removed the refetchData function.